shell/platform/windows/keyboard_win32_unittests.cc

```diff
@@ -1954,57 +1954,6 @@ TEST(KeyboardTest, UpOnlyImeEventsAreCorrectlyHandled) {
   clear_key_calls();
 }
 
-TEST(KeyboardTest, DisorderlyRespondedEvents) {
```

Comment:
Wouldn't you still want to test that events that used to arrive before the framework response no longer do?  Or are the other test changes sufficient to guarantee that?

Reply:
Other test changes are sufficient to guarantee that even when the platform messages arrive too fast for the framework to handle, nothing will be responded out of order and the events turn out to be normal.